test/index_tests.js

```diff
@@ -204,4 +206,39 @@ describe('index.js', () => {
     });
   });
 
+});
+
+describe('index.js with baseUrlPath', () => {
+
+  let fsStub;
+
+  beforeAll( function() {
+    const packageJsonPath = `${process.cwd()}/package.json`;
+    fsStub = sinon.stub(fs, 'existsSync');
+    const packageJson = {
+      name: 'express-oas-generator',
+      version: '1.0.4',
+      description: 'base url',
+      main: 'index.js',
+      baseUrlPath: '/swagger'
+    };
+
+    fsStub.withArgs(packageJsonPath).returns(packageJson);
+  });
+
+  afterAll(function() {
+    fsStub.restore();
+  });
+
+
+  it('WHEN making error request THEN error response should be added to path', () => {
+
+    const app = express();
+    generator.init(app, {});
+    const spec = generator.getSpec();
+    expect(spec.info.description).toBeDefined();
```

Comment:
I think it doesn't test if base URL presented in description or not, may I ask you to improve and actually test two cases when base url is presented and when it is not.

Reply:
fixed